### PR TITLE
Included scraping Recommended products

### PR DESCRIPTION
**Pull request by shubh24**

### CODE REVIEW

File: amazon_rev.py [hunk 1]
```diff
@@ -70,8 +75,37 @@ def doit():
...
+    #Extracting the reviews
+    for i in range(1,6,1): 
+        url_ = amazon_link % (product_asin, i)
+        print "Trying " + url_ + " now!"
+        soup = BeautifulSoup(urlopen(url_).read())
+        for j, row in enumerate(soup('span', {'class': 'review-text'})):
+            li[str(j +10*(i-1))] = row.text
+
+    #Inserting into the Recom collection
+    inserted_review = recom.insert_one(li).inserted_id
```
Comment (psdh) [inline amazon_rev.py]: if you add this segment to the doit function , I think we can reduce the double call to the same url, do I make sense?

File: amazon_rev.py [hunk 2]
```diff
@@ -45,7 +50,7 @@ def doit():
     # will scrape reviews' text
     for j, row in enumerate(soup('span', {'class': 'review-text'})):
         li[str(j + 1)] = row.text
-
+
```
Comment (psdh) [inline amazon_rev.py]: @shubh24 Yaar white spaces hata de :P 
also, can we refactor recom_data to reduce the redundant code ( doit jaise lag raha hai)

File: amazon_rev.py (hunk 1 above)
Comment (shubh24) [inline amazon_rev.py]: Hmm..okay! karta hu

**Comment by psdh** [comment]
testing

**Pull request merged**